components/EditableListItem.tsx

```diff
@@ -37,7 +38,9 @@ export const EditableListItem: React.FC<EditableListItemProps> = props => {
         color={Theme.Colors.Icon}
       />
       <ListItem.Content>
-        <ListItem.Title style={{paddingTop: 3}}>
+        <ListItem.Title
+          {...testIDProps('credentialRegistry')}
```

Comment:
This is common component. Pass testID as prop

Reply:
The list used here is not a custom component. It is used from react-native-elements library.

Reply:
EditableListItem.tsx is a common component.
As of today it's rendering credential registry with two options - mimoto and esignet
It can be reused somewhere else in future